docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -130,20 +109,58 @@ say that the health state and therefore the disability weight is the same for bo
 Impairments
 +++++++++++
 
-In some cases, a medical condition or sequela might have multiple causes. For example, heart failure. 
-It is caused by both IHD and HHD (and others!) and has sequelae associated with both causes. 
-We even learned above that it has the exact same health state in both causes. Heart failure 
-is an example of an impairment. 
+Sometimes, GBD has a lot of information about a health state - such as severe heart failure in the 
+example above - that the modelers want to utilize. However, in the process of normal GBD 
+modeling, this wouldn't be possible. Usually, the cause level information is calculated 
+first and then divded into sequelae. 
 
-An impairment is a sequelae that shows up in 2 or more causes. 
+Therefore, to use this valuable information about a health state, GBD created a new 
+category called impairments. An impairment is medically the same as a sequela, but 
```

Comment:
I think of it as more being medically the same as a health state, although I realize that's not precisely correct either (e.g. the anemia impairment contains multiple health states). Maybe just removing this clause is best?

Reply:
That's one of the things in the official definitions I just can't parse -- does it make sense to you? For example, which sequela is the "anemia" impairment clinically the same as? A whole bunch of them, right?

I think if you understand this and can rephrase it, it could be helpful. But as it is, it _adds_ confusion IMO.

Reply:
Lemme try and rephrase this. Anemia is medically the same as a a bunch of sequelae. I think this is saying that an impairment is also "The medical conditions that can occur among people who contract a disease or suffer an injury." An impairment is medically the same as this definition. If you were a doctor, you'd look at a sequela as the same whether or not it happens to also be an impairment - because it being an impairment is a GBD definition, not a clinical definition.

Reply:
Ah okay, I think I am starting to understand. "An impairment is medically the same as a sequela" means "From a medical perspective, an impairment is similar to a sequela, in that it is a medical condition that can occur among people who contract a disease or suffer an injury." Is that it?

I also don't totally understand what makes a cause different from this. Aren't all three the same from a medical perspective?

Reply:
(Before, I was interpreting it to mean "_each_ impairment is medically the same as _a specific_ sequela".)

Reply:
Yep! That's exactly it. Clinically these are the same, GBD just has these other definitions. 

The second part of your question is more about cause vs sequela. The official difference is the thing "causing" the deaths/disability and the medical condition you experience. So a clear example is diabetes is a cause - it defines what is wrong with your body - and blindness is something you experience as a result of diabetes - a sequela. 

For some of the cardiovascular examples, I think this is a bit fuzzier/harder to understand if you're not a doctor. But its the same idea.

Reply:
Hm -- like I said in the other thread, I'm not convinced. By that logic, I would think anemia is a cause (that is something wrong in the body) and fatigue is a sequela. In the real world, symptoms have causes, and those causes have causes, potentially many layers deep.

I guess I haven't spoken to any cardiologists, though. If they truly would say that IHD and heart failure are totally different things, then I'm probably wrong. But it seems more likely to me that they would say that IHD can cause heart failure which in turn can cause other things.

Reply:
"A medical condition that can occur among people who contract a disease or suffer an injury" seems like it would include anemia, IHD, and heart failure -- I don't see how any of those three fail to meet the definition.

Reply:
I do think there is overlap in these definitions which I know if frustrating. Also - I'm not a doctor and so would have a really hard time trying to define what exactly causes what and what is sequela in a medical sense. 

I guess cause -> medical condition under that cause (sequela) -> symptoms of that medical condition (sometimes overlapping?) 

But I think you can poke holes in this as well and the delineations I do think are often based on a modelers decision of how to define each disease area to make it fit into the GBD framework. Which makes then defining these broadly difficult. If you can think of a good way to include this ambiguity in the page, I am happy to make edits, but I think there are some things here that are how they are because some doctor working for GBD decided it would be so.